media_description.go

```diff
@@ -79,3 +79,14 @@ func (m *MediaName) String() *string {
 	}, " ")
 	return &output
 }
+
+// GetMidValue returns MID value for a media section in SDP
+func (s *MediaDescription) GetMidValue() string {
```

Comment:
I think we should have this in `pion/webrtc` since this just exists to throw away errors? Don't want people using this and being surprised when a missing `mid` doesn't return an error